src/Native/System.Security.Cryptography.Native/pal_ssl_extra_cert.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_types.h"
+
+#include <openssl/ssl.h>
+
+/*
+Function:
+AddExtraCertToChain
+
+adds certificate to the extra chain certificates associated with ssl_ctx
+
+libssl frees the x509 object.
+Returns 1 if success and 0 in case of failure
+*/
+extern "C" int AddExtraChainCertToSslCtx(SSL_CTX* ssl, X509* x509);
+
```

Comment:
Can you make sure to run ~corefx/src/Native/format-code.sh on the native code before it gets merged?  This ensures the native code is formatted correctly (and when the next person edits it, they don't need to deal with a different authors changes).